api/catalog/api/controllers/search_controller.py

```diff
@@ -168,17 +168,15 @@ def _apply_filter(
     :return: the input ``Search`` object with the filters applied
     """
 
-    if serializer_field in search_params.data:
-        filters = []
-        for arg in search_params.data[serializer_field].split(","):
-            _param = es_field or serializer_field
-            args = {"name_or_query": "term", _param: arg}
-            filters.append(Q(**args))
-        method = getattr(s, behaviour)
-        return method("bool", should=filters)
-    else:
+    arguments = search_params.data.get(serializer_field)
+    if arguments is None:
         return s
 
+    parameter = es_field or serializer_field
```

Comment:
```suggestion
   arguments = arguments.split(',')

    parameter = es_field or serializer_field
```

Reply:
The `terms` query below expects the `parameter` to be a list, not a string. If you split the `arguments` string by a comma (this way, `'by,by-sa'` becomes `['by', 'by-sa']`, for example), it should work.